CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
```

Comment:
I appreciate that we're in the early days of rolling out a carefully-considered IOG-wide approach to Open Source. But as a project maintainer of this repository, I'll feel unprepared for this important responsibility until I get some information/explanations about IOG's more detailed intentions/SOPs/etc regarding eg what is "appropriate and fair corrective action", what evidence do I need before acting, etc.

Reply:
Hmm. Or maybe I'm not actually a _project maintainer_ in this context, and my only responsibility as a "repo owner" would be to contact `code-of-conduct@` as discussed in `## Enforcement` below?

Reply:
Would appreciate any clarification you could provide for the questions I raised in this thread, @Trueblueaddie . Thanks!

Reply:
I think the implementation of a CoC, as a first step, is an indication of IOG's commitment to creating a healthy open source community. As far as a SOP goes, there are a few conversations going on with legal and HR - nothing definitive yet. Training is also being discussed to support this new SOP.  

The definition of roles and responsibilities is a critical point. In the documentation request, is an ask for the identification of the core maintainers typically listed in the README.md. It would be good to have an IOG standard for the definition of  a  "repo owner", "project maintainer", "core maintainer", etc.